src/test/java/org/simplify4u/plugins/KeyInfoTest.java

```diff
@@ -37,7 +39,7 @@
                 {"0x123456789abcdef0,0x0fedcba987654321", 0x123456789abcdef0L, true},
                 {"0x123456789abcdef0, 0x0fedcba987654321", 0x123456789abcdef0L, true},
                 {"0x123456789abcdef0", 0x231456789abcdef0L, false},
-                {"0x123456789abcdef0, 0x0fedcba987654321", 0x321456789abcdef0L, false}
+                {"0x123456789abcdef0, 0x0fedcba987654321", 0x321456789abcdef0L, false},
```

Comment:
comma not required

Reply:
True. Remnant of an earlier change. I'll undo this change.